### ISSUE TITLE
search by OMOP concept ID

### ISSUE THREAD

**Issue by vojtechhuser** [OHDSI/Hermes#17]
I am trying to find CPT concept

CDMV:43527963	name:Unlisted multianalyte assay with algorithmic analysis	CPT:81599

for example this way:
http://www.ohdsi.org/web/hermes/index.html#/concept/43527963

Why nothing is found?

also searching for "multianalyte" finds no hits.

Those concepts *were* in CDMV v4.

Other missing are:

CDMV,name,CPT
43527955	Unlisted molecular pathology procedure	81479
43527956	Oncology (ovarian), biochemical assays of two proteins (CA-125 and HE4), utilizing serum, with menopausal status, algorithm reported as a risk score	81500
43527957	Oncology (ovarian), biochemical assays of five proteins (CA-125, apoliproprotein A1, beta-2 microglobulin, transferrin, and pre-albumin), utilizing serum, algorithm reported as a risk score	81503
43527958	Endocrinology (type 2 diabetes), biochemical assays of seven analytes (glucose, HbA1c, insulin, hs-CRP, adoponectin, ferritin, interleukin 2-receptor alpha), utilizing serum or plasma, algorithm reporting a risk score	81506
43527959	Fetal congenital abnormalities, biochemical assays of two proteins (PAPP-A, hCG [any form]), utilizing maternal serum, algorithm reported as a risk score	81508
43527960	Fetal congenital abnormalities, biochemical assays of three proteins (PAPP-A, hCG [any form], DIA), utilizing maternal serum, algorithm reported as a risk score	81509
43527961	Fetal congenital abnormalities, biochemical assays of three analytes (AFP, uE3, hCG [any form]), utilizing maternal serum, algorithm reported as a risk score	81510
43527962	Fetal congenital abnormalities, biochemical assays of four analytes (AFP, uE3, hCG [any form], DIA) utilizing maternal serum, algorithm reported as a risk score (may include additional results from previous biochemical testing)	81511
43528050	Fetal congenital abnormalities, biochemical assays of five analytes (AFP, uE3, total hCG, hyperglycosylated hCG, DIA) utilizing maternal serum, algorithm reported as a risk score	81512
43527963	Unlisted multianalyte assay with algorithmic analysis	81599

**Comment by leeevans** [comment]
CPT (version 4) is one of the restricted vocabularies. Only the unrestricted vocabularies can be made publicly available.  Therefore there are no CPT codes included in the publicly accessible installation of HERMES.

**Comment by cgreich** [comment]
Which brings up a good point: As soon as we have the permission to show folks after a click-through "I agree" page, it could be made available. CPT is very core to what we are doing, so we should consider that.

**Comment by fdefalco** [comment]
I would happily implement that 'i agree' page.  What is it we have to receive agreement on?  Do we need to keep record of anything for legal purposes?

**Comment by cgreich** [comment]
Yes, we do. We are implanting this as we speak in the download page. And I am arguing with the AMA. But whatever the text is, you can already prepare some yadayada.
